src/steps/02-EvaluationCriteria/JandA/ProcurementDiscussion.vue

```diff
@@ -15,9 +61,104 @@
 
 <script lang="ts">
 import Vue from "vue";
-import { Component } from "vue-property-decorator";
+import { Component, Mixins } from "vue-property-decorator";
+import AcquisitionPackage from "@/store/acquisitionPackage";
+import { getYesNoRadioOptions, hasChanges } from "@/helpers";
+import _ from "lodash";
+import ATATTextArea from "@/components/ATATTextArea.vue";
+import ATATRadioGroup from "@/components/ATATRadioGroup.vue";
+import { RadioButton } from "../../../../types/Global";
+import { FairOpportunityDTO } from "@/api/models";
+import SaveOnLeave from "@/mixins/saveOnLeave";
+
+@Component({
+  components:{
+    ATATTextArea,
+    ATATRadioGroup
+  }
+})
+
+export default class ProcurementDiscussion extends Mixins(SaveOnLeave) {
+  public procurementParagraphText = ""
+  public procurementText = "";
+  public suggestedImpactText = ""
+  public procurementImpact = "";
+  public cspName = ""
+  public existingEnv = ""
+  public existingEnvOptions: RadioButton[] = getYesNoRadioOptions("existingEnvOptions")
+
+  private get currentData(): FairOpportunityDTO {
+    return {
+      /* eslint-disable camelcase */
+      procurement_discussion: this.procurementText,
+      procurement_has_existing_env: this.existingEnv,
+      procurement_previous_impact: this.procurementImpact
+    } as FairOpportunityDTO;
+  }
+
+  private get savedData(): FairOpportunityDTO {
+    return {
+      procurement_discussion: AcquisitionPackage.fairOpportunity?.procurement_discussion || "",
+      procurement_has_existing_env: AcquisitionPackage.fairOpportunity?.
+        procurement_has_existing_env || "",
+      procurement_previous_impact: AcquisitionPackage.fairOpportunity?.
+        procurement_previous_impact || "",
+    } as FairOpportunityDTO;
+  }
+
+  private hasChanged(): boolean {
+    return hasChanges(this.currentData, this.savedData);
+  }
+
+  public async loadOnEnter(): Promise<void> {
+    const storeData = _.cloneDeep(AcquisitionPackage.fairOpportunity);
+    if (storeData) {
+      const farSelected = storeData.exception_to_fair_opportunity
+      if(farSelected === "YES_FAR_16_505_B_2_I_A"){
+        this.procurementParagraphText = "Identify any schedule requirements, unique" +
+          " features and mandatory requirements, or the existence of" +
+          " proprietary data, copyrighted information or a patent which" +
+          " restrict competition. Since you cited the “unusual and compelling" +
```

Comment:
If selected `YES_FAR_16_505_B_2_I_C` nothing is showing. But since the text is the same up to `...restrict competition.` -- set that as the default, and only add the `Since you cited the...` text if `farSelected === "YES_FAR_16_505_B_2_I_A"`